src/integrations/Twilio.ts

```diff
@@ -0,0 +1,66 @@
+import {IntegrationName, IntegrationPurpose, IQualifierIntegration, IVerificationContext} from '../types';
+
+import { IObserverIntegration } from './../types';
+import Integration from "./Integration";
+
+export interface ITwilioConfig {
+  serviceSid: string;
+  authToken: string;
+}
+
+export interface ITwilioProps {
+  url: string;
+  config: ITwilioConfig;
+  requestId: string;
+}
+
+export default class TwilioIntegration extends Integration implements IQualifierIntegration{
```

Comment:
Does this integration actually do anything / need to be loaded on the client-side for 2FA to work?